cms_templates/admin.py

```diff
@@ -325,12 +325,95 @@ def clean(self):
         return cleaned_data
 
 
+def get_templates_that_use_template(template_name, only_one_required=False):
+    quoted_name = "'%s'" % template_name
+    double_quoted_name = '"%s"' % template_name
+
+    candidates = Template.objects.filter(Q(content__contains=quoted_name) |
+                                         Q(content__contains=double_quoted_name))
```

Comment:
If this query is slow, you could do a two pass lookup where first you look for `candidates = Template.objects.filter(content__contains=template_name)` and then you check for single/double quotes in the subset.